src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -556,11 +556,15 @@ CreateDistributedTable(Oid relationId, Var *distributionColumn, char distributio
 	{
 		List *partitionList = PartitionList(relationId);
 		Oid partitionRelationId = InvalidOid;
+		char *parentRelationName, *nspname;
```

Comment:
Our usual style (which is slightly different from PG) is to have declaration and assignment be the same line, and to avoid acronyms.

e.g.
```c
char *schemaName = get_namespace_name(get_rel_namespace(relationId));
char *parentRelationName = ...;
```

We also tend to avoid nested calls to explicitly name the arguments, so the usual pattern is:
```c
Oid namespaceId = get_rel_namespace(relationId);
char *schemaName = get_namespace_name(namespaceId);
```

Reply:
Changed to the new format.